crates/dataflow-jit/src/codegen/call.rs

```diff
@@ -100,6 +103,16 @@ impl CodegenCtx<'_> {
         }
     }
 
+    fn error_abort(&self, builder: &mut FunctionBuilder<'_>) {
+        let trap = builder.ins().trap(TRAP_ABORT);
+
+        if let Some(writer) = self.comment_writer.as_deref() {
+            writer
+                .borrow_mut()
+                .add_comment(trap, "call @dbsp.error.abort()");
```

Comment:
in the future we should add information to help users identify where the error happened. For example, having some kind of error message from the compiler could work.

Reply:
Yah, I'll get around to that